WcaOnRails/config/wca-states.json

```diff
@@ -1550,67 +1598,9 @@
           "id": "Zimbabwe"
         }
       ],
-      "title": "UN Member States:"
-    },
-    {
-      "states": [
-        {
-          "info": "",
-          "name": "Holy See",
-          "continent_id": "_Europe",
-          "class": "state",
-          "iso2": "VA",
-          "id": "Holy See"
-        },
-        {
-          "info": "",
-          "name": "Palestine",
-          "continent_id": "_Asia",
-          "class": "state",
-          "iso2": "PS",
-          "id": "Palestine"
-        }
-      ],
-      "title": "UN Permanent Observer States:"
-    },
-    {
-      "states": [
-        {
-          "info": "A fully self-governing autonomous region and was represented as the Republic of China (ROC) at the UN until 1971 when the People's Republic of China (PRC) assumed China's seat.\nCurrently has recognition as a country by 20 UN member states.\n",
-          "name": "Taiwan",
-          "continent_id": "_Asia",
-          "class": "state",
-          "iso2": "TW",
-          "id": "Taiwan"
-        },
-        {
-          "info": "Declared independence from Serbia in February 2008.\nCurrently has recognition as a country by 115 UN member states as of February 27th, 2017.\n",
-          "name": "Kosovo",
-          "continent_id": "_Europe",
-          "class": "state",
-          "iso2": "XK",
-          "id": "Kosovo"
-        },
-        {
-          "info": "Is formally known as Hong Kong Special Administrative Region of the People's Republic of China. It is a fully autonomous region and maintains a separate political and economic system from China aside from its foreign affairs.\nDue to Hong Kong's long history in the WCA and large community, it is recognized as a separate entity to China within the WCA.\n",
-          "name": "Hong Kong",
-          "continent_id": "_Asia",
-          "class": "state",
-          "iso2": "HK",
-          "id": "Hong Kong"
-        },
-        {
-          "info": "Is formally known as Macao Special Administrative Region of the People's Republic of China.\nIt is functionally identical to Hong Kong, as so we treat it the same way as Hong Kong within the WCA.",
-          "name": "Macau",
-          "continent_id": "_Asia",
-          "class": "state",
-          "iso2": "MO",
-          "id": "Macau"
-        }
-      ],
-      "title": "The following countries are non-UN member states recognized by the WCA:"
+      "title": ""
     }
   ],
   "version": "Version: June 19, 2017",
```

Comment:
What is this key used for? Should it have also gotten bumped?

Reply:
Nice catch, it totally should have!
See thewca/wca-regulations#622.